Automattic/jetpack
# Facebook Likebox: adapt_container_width

#### Changes proposed in this Pull Request:

* Fixes this: https://github.com/Automattic/jetpack/issues/9889
* Allows input of `0`
* Input of `0` adds `data-adapt-container-width="true"` to likebox 
* Input of anything between `0` and `180` (the minimum width suggested by [FB](https://developers.facebook.com/docs/plugins/page-plugin/)) will set the width to `default_width` and will add `data-adapt-container-width="false"` 
* Added description of what will happen if you enter 0
* Added some `<br />` elements to make things look better (a little more spaced out) 

#### Testing instructions:






* Add the Likebox widget
* Set the value to 0 and save. Go to the front-end and inspect the likebox. You will see `data-adapt-container-width="true"`
* Go back, set the value to anything between 0 and 180, it will default to default_width that's 340. Check the frontend, you will see `data-adapt-container-width="false" data-width="340"`
* Set the value b/w min and max width, it works as expected (the same as before this PR).

#### Proposed changelog entry for your changes:

* Adaptive width support for Facebook Likebox widget
